app/src/org/commcare/android/database/global/DatabaseGlobalOpenHelper.java

```diff
@@ -27,7 +27,7 @@
      * V.3 - ApplicationRecord has new fields to support multiple app seating, FormsProvider
      * and InstanceProvider use per-app databases
      */
-    private static final int GLOBAL_DB_VERSION = 3;
+    public static final int GLOBAL_DB_VERSION = 3;
```

Comment:
...Was at one point trying to do something useless with it and reverted, forgot to change back. Thanks for catching.